src/python/gudhi/alpha_complex.pyx

```diff
@@ -53,10 +53,12 @@ cdef class AlphaComplex:
 
     """
 
-    cdef Alpha_complex_interface * thisptr
+    cdef Alpha_complex_interface * this_ptr
+    cdef bool fast
+    cdef bool exact
 
     # Fake constructor that does nothing but documenting the constructor
-    def __init__(self, points=None, off_file=''):
+    def __init__(self, points=None, off_file='', complexity='safe'):
```

Comment:
I see why you would call it complexity, but I am not too fond of the name. Precision? I am open to other suggestions, or even to keep complexity if someone else likes it.

Reply:
I am ok with your proposal, I renamed it precision on 4f02ba9c